.qi.yml

```diff
@@ -0,0 +1,20 @@
+app_name: gpii-handlebars
```

Comment:
Can you comment these new mysterious files in order to explain their purpose and provenance?